### PR TITLE
STAF-26: Add language selector to login page

### PR DESCRIPTION
**Pull request by RomanLavrunov**

### CODE REVIEW

File: src/core/components/LanguageSwitcher/LanguageSwitcher.tsx [hunk 1]
```diff
@@ -1,16 +1,54 @@
...
+      <Select
+        MenuProps={menuProps}
+        className="language-switcher-select-standard"
+        value={selectedLanguage}
+        label={selectedLanguage}
+        onChange={handleLanguageChange}
+        disableUnderline={true}
+        IconComponent={PublicIcon}
+      >
+        {Object.keys(LANGUAGE).map((code: string) => (
```
Comment (devmc-ee) [inline src/core/components/LanguageSwitcher/LanguageSwitcher.tsx:41]: we would simplify this statement and the next ones, if we could export an array of lowercased languages from ../../../config/i18n

File: src/pages/Login/Login.tsx [hunk 2]
```diff
@@ -11,6 +12,7 @@ export const Login = () => {
     <Page classes="login">
       <div className="login-page__app-title">{title}</div>
       <div className="login-page__form-container">
+        <LanguageSwitcherContainer></LanguageSwitcherContainer>
```
Comment (devmc-ee) [inline src/pages/Login/Login.tsx:15]: Please use a single self closed tag, if it doesn't contains children:
```<LanguageSwitcherContainer/>```

File: src/pages/Login/Login.tsx [hunk 3]
```diff
@@ -2,6 +2,7 @@ import { useTranslation } from 'react-i18next';
 import { Page } from '../../core/components/Page';
 import { LoginForm } from '../../user/components/LoginForm';
 import './LoginPage.css';
+import { LanguageSwitcherContainer } from '../../core/components/LanguageSwitcher/LanguageSwitcherContainer';
```
Comment (devmc-ee) [inline src/pages/Login/Login.tsx:5]: Container is a kind of system component (controller, wrapper). At the end, we want to use a component by itself. And we dont care if it is a wrapper or container or pure presentational component. So, if we have containers and components in the same folder (that should have the name same as the component), lets re-export containers with the name of component. It makes cleaner code and points the usage of component

To do so, please add an index.ts file to the` ../../core/components/LanguageSwitcher ` folder, and make re-export:
```typescript
export { LanguageSwitcherContainer as LanguageSwitcher } from './LanguageSwitcherContainer';
```
Then import can be simplified:
````typescript
import { LanguageSwitcher } from '../../core/components/LanguageSwitcher';

**Review by devmc-ee — CHANGES_REQUESTED** [review]
Beside minor changes requested in the comments, please add stories for storybook.

File: src/config/i18n.ts [hunk 4]
```diff
@@ -8,6 +8,12 @@ import en from '../assets/translations/en.json';
 import et from '../assets/translations/et.json';
 import th from '../assets/translations/th.json';
 
+export const LANGUAGE: { [key: string]: { [key: string]: unknown } } = {
```
[the remark this answers is not in the document]
Comment (RomanLavrunov) [reply src/config/i18n.ts]: Fixed! Added enum and an array of languges, which will be exported to the component! Thanks!

File: src/config/i18n.ts [hunk 5]
```diff
@@ -17,13 +23,13 @@ i18n
     fallbackLng: 'en',
     resources: {
       en: {
-        translation: en,
+        translation: LANGUAGE.EN,
```
[the remark this answers is not in the document]
Comment (RomanLavrunov) [reply src/config/i18n.ts]: Not quite sure if there's any need for that at the moment. I reverted all changes and it works just perfectly fine at the moment.

File: src/core/components/LanguageSwitcher/LanguageSwitcher.tsx (hunk 1 above)
Comment (RomanLavrunov) [reply src/core/components/LanguageSwitcher/LanguageSwitcher.tsx]: Thanks for the hint! It works as you suggested now.

File: src/pages/Login/Login.tsx (hunk 2 above)
Comment (RomanLavrunov) [reply src/pages/Login/Login.tsx]: Fixed!

File: src/pages/Login/Login.tsx (hunk 3 above)
Comment (RomanLavrunov) [reply src/pages/Login/Login.tsx]: I changed the name in index file. Seems to be working ok.

File: src/core/components/LanguageSwitcher/LanguageSwitcher.stories.tsx [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+  };
+
+  const languageSwitcherProps = {
+    ...args,
+    language: value,
+    onChange: handleLanguageChange,
+  };
+
+  return (
+    <>
```
Comment (devmc-ee) [inline src/core/components/LanguageSwitcher/LanguageSwitcher.stories.tsx:27]: if we return only one single component, then we dont need to wrap if with fragments. So, the return can be simplified to 
```javascript 
return <LanguageSwitcher {...languageSwitcherProps} />;
```

**Review by devmc-ee — CHANGES_REQUESTED** [review]

File: src/core/components/LanguageSwitcher/LanguageSwitcher.stories.tsx (hunk 6 above)
Comment (RomanLavrunov) [reply src/core/components/LanguageSwitcher/LanguageSwitcher.stories.tsx]: Okido, fixed it!

**Review by devmc-ee — APPROVED** [review]

**Pull request merged**